src/Datadog.Trace/ClrProfiler/InstrumentationDefinitions.cs

```diff
@@ -0,0 +1,235 @@
+// <copyright file="InstrumentationDefinitions.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+namespace Datadog.Trace.ClrProfiler
+{
+    internal static class InstrumentationDefinitions
+    {
+        internal static NativeCallTargetDefinition[] GetAllDefinitions()
+        {
+            return new NativeCallTargetDefinition[]
+            {
+                new("AerospikeClient", "Aerospike.Client.AsyncCommand", "ExecuteCommand",  new[] { "System.Void" }, 4, 0, 0, 4, 65535, 65535, "Datadog.Trace, Version=1.28.6.0, Culture=neutral, PublicKeyToken=def86d061d0d2eeb", "Datadog.Trace.ClrProfiler.AutoInstrumentation.Aerospike.AsyncCommandIntegration"),
```

Comment:
I think I know the answer, but figured I'd ask anyways. I noticed that there are no longer integration names for each integration. Is that because the instrumentation methods are supposed to be checking whether the integration is enabled or not?

Reply:
Yes, right now we are doing the check in the managed side (eg. https://github.com/DataDog/dd-trace-dotnet/blob/master/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AWS/SQS/AwsSqsCommon.cs#L27). But now you mention this, maybe we can improve it and move it to this instrumentation definitions file by filtering the array before returning (or avoiding the instance generation), so we avoid the rewriting completely. That will reduce memory (ducktyping and dynamic methods to run the integration).  But maybe that's outside this PR? WDYT?

Reply:
Yeah I think that is out of scope for this PR. However we decide to implement it, I think it would be helpful to have a built-in mechanism to filter out integrations so developers don't need to manually write the check `if (tracer.Settings.IsIntegrationEnabled())`